File: Data Structure/Lab 3/AvgArr.c
// Write a program to calculate average of first n numbers. 

#include<stdio.h>
void main(){
    int n,arr[n],i=0;
    int sum = 0;
    float avg = 0;

    printf("Enter size of an array(n) :");
    scanf("%d",&n);

    for(i = 0; i < n ; i++){
        printf("Enter element :");
        scanf("%d",&arr[i]);
    }

    for(i = 0; i < n ; i++){
        sum = sum + arr[i];
    }

    avg = (float) sum / n;

    printf("Avrage of array element is : %f ",avg);
}

File: Data Structure(Java)/Lab 2/AvgArr.java
//Write a program to calculate average of first n numbers.

import java.util.Scanner;
public class AvgArr {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter n :");
        int n = sc.nextInt();
        int[] arr = new int[n];
        int sum = 0;
        float avg = 0;

        for(int i = 0; i < n; i++){
            System.out.println("Enter number at "+i);
            arr[i] = sc.nextInt();
        }

        for(int i = 0; i < n; i++){
            sum += arr[i];
        }

        avg = sum / n;

        System.out.println(avg);

        sc.close();
    }
}

